src/app/archive-settings/manage-metadata/subcomponents/form-edit/form-edit.component.html

```diff
@@ -0,0 +1,43 @@
+<div #value class="value">
+  <div
+    class="value-line"
+    *ngIf="!editing; else valueEditor"
```

Comment:
The reason is just the order in which I wrote the various "modes" of the editors. I can change it here to be consistent (as well as to use the `editing` value affirmatively rather than checking for its negation)

Reply:
:+1: I think it's worthwhile for this to be consistent with `form-create`